README.md

```diff
@@ -126,11 +147,11 @@ To begin benchmarking your first embodied agent with NETT, please be aware:
 
 - **Environment**
 
-   Create the simulation environment using the path to your Unity executable (see Step 1).
+   Create the simulation environment using the path to your Unity executable (see Step 1). The path must be absolute and not relative.
```

Comment:
That is incorrect. The path can be relative.

Reply:
That's even easier then! Please feel free to change that.